src/common/main.py

```diff
@@ -12,6 +12,7 @@
 import mido
 import mido.backends.rtmidi  # Needed for windows builds w/ pyinstaller
 
+CONSOLE_MODE = True # Flip to enable devleoping in console mode

```

Comment:
Maybe change the comment on this a bit so people realize this doesn't disable the UI, this took me a sec to process.